web-components/src/components/phone-input/PhoneInput.ts

```diff
@@ -49,6 +51,21 @@ export namespace PhoneInput {
       });
     }
 
+    attributeChangedCallback(name: string, oldval: any, newval: any) {
+      super.attributeChangedCallback(name, oldval, newval);
+      if(name.toLowerCase() === 'codePlaceholder'.toLowerCase()) {
+        this.flagClass = newval === '+1' ? 'flag__usa' : '';
```

Comment:
We want to move away from any explicit country selection, have it respond to user settings for their own Phone Input usage